lib/improver/tests/acceptance/test_temporal_interpolate.py

```diff
@@ -51,37 +51,23 @@ def test_basic(tmp_path):
     output_path = tmp_path / "output.nc"
     args = [*input_paths,
             "--times", "20190116T1000Z",
-            "--output_files", output_path]
+            "--output", output_path]
     run_cli(args)
     acc.compare(output_path, kgo_path)
 
 
 def test_linear_interval(tmp_path):
```

Comment:
This test is currently failing because we don't have any new kgo for the change to saving a cubelist rather than separate files from each cube. I've looked at the output the pytest saves and it looks ok, but I didn't know if we want to save it in a cube or a cubelist? Iris will load this new file as a cubelist (with an extra cube for the prefixes) but both improver load_cube and load_cubelist both merge the multiple timesteps into a single cube. What do you think?

Reply:
Yep - I put on the top of the PR that I'd need to create the new KGO.

On saving a cube vs cubelist - my gut instinct is that we probably want to save a cube since the nowcasting extrapolate CLI (which breaks the "one timestep per file" convention) creates a single cube - either way, it's a simple fix to revert if it turns out we want a cubelist.

Reply:
Now done, generated new KGO, but will put this in when the final PR to master is created.